resolwe/flow/migrations/0011_calculate_checksum.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2016-10-13 09:15
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from resolwe.flow.utils import get_data_checksum
+
+
+def calculate_checksum(apps, schema_editor):
+    # We can't import the Person model directly as it may be a newer
```

Comment:
What `Person` model? And why this comment, this is something that is known when doing migrations as you should always use historic versions of models in migrations.